local/Vagrantfile

```
@@ -2,7 +2,7 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "debian/bullseye64"
+  config.vm.box = "debian/bookwarm64"
```

Le nom de l'image n'est pas le bon dans le fichier `Vagrantfile` de la branche `main`
```suggestion
  config.vm.box = "debian/bookworm64"
```
C'est problématique seulement pour l'exécution via virtualbox comme provider Vagrant, pas de soucis avec docker 🙂

---

Merci Willy 🙏